modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -8,8 +8,6 @@
 
 An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
 
-`ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
```

Comment:
I am putting it back for now before deciding what to do next :+1:

Reply:
Suggestion:

`Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
`

Reply:
Note that the use of `ClusterTasks` is not compliant with the current OpenShift documentation standards/ As we are changing this paragraph we must make it compliant or it will fail peer/merge review.

Reply:
I implemented the changes as suggested. I really like that solution.